src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -310,6 +310,27 @@ function getApplePayConfig() {
   };
 }
 
+function getKlarnaConfig() {
+  const {klarnaWidgetEnabled} = window;
```

Comment:
`{ klarnaWidgetEnabled }`

Reply:
Do we have this in ESLint?

Reply:
This will be fixed in another PR since the branch name is also not correct.